common/lanelet2_extension/CMakeLists.txt

```diff
@@ -88,88 +115,122 @@ else() # Default to ROS 1 logging
   
 endif()
 
-find_package(catkin REQUIRED COMPONENTS
-  ${deps_using_buildtool}
-)
 
-message("Logger includes: " ${${LOGGER_DEP}_INCLUDE_DIRS})
-message("Logger Libs: " ${${LOGGER_DEP}_LIBRARIES})
+if(${ROS_VERSION} EQUAL 2)
```

Comment:
Can we not just use the set variable ROS_VERSION?

Reply:
This is the pattern we have been using for all cmake variables. I don't think it was causing issues before, but I'm trying to keep things uniform.